### DISCUSSION TITLE
পুল রিকোয়েস্ট পরবর্তি কিছু কাজ

### DISCUSSION THREAD

**Discussion by zonayedpca** [devsonket/devsonket.github.io#582] [ঘোষণা]
আমাদের সিস্টেম কিছুটা অটোম্যাটেড করা আছে, তাই কোনো পুল রিকোয়েস্টে যদি ভুল থাকে তাহলে সেটার একটা ম্যাসেজ দেখতে পাবেন এরকমঃ 
![image](https://user-images.githubusercontent.com/18544717/136710161-314f9398-6e2d-413c-bff6-3d6f15fd1d36.png)

যদিও আমাদের মেইন্টেইনাররা আপনাকে ভুল সংশোধন করার জন্য নোটিফাই করবে, বাট দ্রুত পুল রিকোয়েস্ট অ্যাক্সেপ্ট করানো জন্য এই ম্যাসেজ দেখামাত্র ভুলটা বের করে নিজেই সংশোধন করে ফেলতে পারেন। বেশিরভাগ ক্ষেত্রেই দেখা যায় জেসন ফাইলের ফরম্যাটগত ভুল হয়েছে, সেক্ষেত্রে একটা ভালো [জেসন ভ্যালিডেটর](https://jsonlint.com/) অনলাইন থেকে খুঁজে বের করে আপনার ফাইলটা ভ্যালিডেট করে দেখতে পারেন। ভুল থাকলে সেটা কোথায় হয়েছে সেটা দেখাবে, আপনি জাস্ট ফিক্স করে পুশ করলেই আশা করি গ্রীন সিগন্যাল দেখতে পাবেন। ঠিক তখনি আপনার পুল রিকোয়েস্ট মার্জের জন্য রেডি বলে ধরে নিতে পারবেন... 
![image](https://user-images.githubusercontent.com/18544717/136710271-505c6929-f38b-41dd-9e8d-0981a3e78167.png)

ধন্যবাদ...

**Comment by asifadib** [comment]
এইটা কিছুটা হলেও মেইন্টেইনারদের কাজ কমাবে। নিজের ভুল নিজেই খুঁজে বেড় করতে পারবো এখন থেকে ইনশাল্লাহ।
